pkg/kustomize/secrets.go

```diff
@@ -52,7 +52,9 @@ func FieldGroupFor(
 
 		fieldGroup.FeatureSecurityScanner = true
 		fieldGroup.SecurityScannerV4Endpoint = fmt.Sprintf(
-			"http://%s-clair-app:80", quay.GetName(),
+			"http://%s-clair-app.%s.svc.cluster.local",
```

Comment:
I realize you didn't introduce this, but is there a better way to get this address? It feels fragile.

Reply:
What do you suggest?

Reply:
Yeah that's the thing, I don't really know a better way either.

Reply:
AFAIK the domain "cluster.local" always exist in a cluster, the host "svc" in the domain indicates that it is a service, then we have the namespace and the service name:

```
<servicename>.<namespace>.svc.cluster.local
```

Host "svc.cluster.local" is automatically added to the NO_PROXY list. @dmage do you know if this domain can be customized to something else?

Reply:
As discussed in our Operators meeting we can keep this hardcoded, therefore I am resolving this one.

/cc @dmage @flavianmissi